O365/sharepoint.py

```diff
@@ -185,6 +186,23 @@ def get_lists(self):
         data = response.json()
 
         return [self.list_constructor(parent=self, **{self._cloud_data_key: lst}) for lst in data.get('value', [])]
+    
+    def get_listbyname(self,displayName):
```

Comment:
```suggestion
    def get_list_by_name(self,displayName):
```
We are using [pep8](https://www.python.org/dev/peps/pep-0008/#function-and-variable-names) as our style guide. That requires every word in a function name to be separated by an underscore.